rotkehlchen/chain/ethereum/adex/adex.py

```diff
@@ -1033,16 +1049,12 @@ def get_events_history(
             )
 
         identity_address_map = self._get_identity_address_map(addresses)
-        try:
-            staking_events = self._get_staking_events(
-                addresses=addresses,
-                identity_address_map=identity_address_map,
-                from_timestamp=from_timestamp,
-                to_timestamp=to_timestamp,
-            )
-        except DeserializationError as e:
```

Comment:
So after these changes _get_staking_events can no longer raise DeserializationError?

Reply:
Exactly, as it is handled by `_get_new_staking_events_graph()` (via `_get_staking_events_graph()`) and re-raised as RemoteError.